### PR TITLE
Toolbar & Headline Section (Mobile Vesrion)

### PR DESCRIPTION
**Pull request by umair-rao**
I have choosed template No. 3 and coded HTML and CSS file using Flexbox for the first two section which are below.
1. Toolbar Section
2. Heading Section

Please review and let me know if changings are required.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <title>Mobile Vesrion</title>
+</head>
+<body>
+<div class="container">
+    <div class="toolbar">
+        <p id="welcome">Welcome</p>
+        <div class="menu">
+            <img src="images/menu.svg" alt="menu_icon">
+        </div>
+        </div>
```
Comment (MaganaAsiati) [inline index.html]: - [x] Kindly make sure the container of this section is a `header `tag. 
Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. :grinning:  :grinning:  :grinning: 

- [x] The logo (if the design has one) is on the left and has a link (`a` tag).
   * This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.  :grinning:  :grinning:  :grinning: 

- [x] The hamburger button is on the right and has a `nav` tag.
   * The `nav` tag is for semantics reasons, and the right position for convention and user experience for mobile devices.  :grinning:  :grinning:  :grinning: 

 - check the example below :+1: 

```
<header>
        <a href="index.html">
          <img src="images/logo.png" alt="Logo" />
        </a>

        <nav class="hamburger-menu">
          <div class="bar-top"></div>
          <div class="bar-middle"></div>
          <div class="bar-bottom"></div>
        </nav>
    </header>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <div class="headline_section">
```
Comment (MaganaAsiati) [inline index.html]: - The container of the section is a `section` tag. Kindly  replace the `<div>` tag  by the `<section> `tag :grinning: :grinning: :grinning: 

   * It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        </div>
+        </div>
+        <div class="link">
+            <img src="images/twitter.svg" alt="twitter_icon">
+            <img src="images/Linkedin icon.svg" alt="Linkedin_icon">
+            <img src="images/medium.svg" alt="medium_icon">
+            <img src="images/github.svg" alt="github_icon">
+            <img src="images/angellist.svg" alt="angellist_icon">
+        </div>
+    </div>
```
Comment (MaganaAsiati) [inline index.html]: - Kindly for the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).:grinning: :grinning: :grinning: 

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

check the example below :+1: 
```
<ul>
            <li> <a href=""><img src="images/twitter.png" alt="Twitter logo" /></a></li>
            <li><a href=""><img src="images/twitter.png" alt="Twitter logo" /></a></li>
            <li><a href=""><img src="images/git.png" alt="Git logo" /></a></li>
            <li><a href=""><img src="images/m.png" alt="M logo" /></a></li>
            <li><a href=""><img src="images/nice.png" alt="V logo" /></a></li>
</ul>
```

File: style.css [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  font-size: 20px;
+  align-items: center;
+  text-align: center;
+}
+
+.link {
+  align-content: center;
+  justify-content: space-around;
+  padding-left: 35%;
+}
```
Comment (MaganaAsiati) [inline style.css]: - Kindly make sure your icons are showing properly and the design of your icons looks like the Figma design as required in the project, follow this link to Check the [Figma design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)Check the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

- Check below your design compared to the Figma design :grinning: :grinning: :grinning: 

## Figma  design :+1: 

image

## Your design :-1: 

image

File: style.css [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+.second-text {
+  width: 327px;
+  height: 164px;
+  font-weight: 400;
+  font-size: 20px;
+  align-items: center;
+  text-align: center;
+}
```
Comment (MaganaAsiati) [inline style.css]: - Kindly ensure that the color, font size, line height, and font weight of the introductory text match the Figma design you chose. You can check out the Figma design specification for the exact properties used👍🏽

## Your design

image

## Figma design 

image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @umair-rao 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (umair-rao) [reply index.html]: All changes are made as suggested. Please check if any is still missing, please let me know.

File: index.html (hunk 2 above)
Comment (umair-rao) [reply index.html]: Replaced with section.

File: index.html (hunk 3 above)
Comment (umair-rao) [reply index.html]: Changes are made as suggested. Please check.

File: style.css (hunk 4 above)
Comment (umair-rao) [reply style.css]: Designed as per Figma design. Please check.

File: style.css (hunk 5 above)
Comment (umair-rao) [reply style.css]: Designed as per Figma design. Please check.

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle:
success

## Hey @umair-rao :wave:

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

:heavy_check_mark: Following the GitHub Flow
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Descriptive PR Title

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**